cmd/juno/cli/root.go

```diff
@@ -8,25 +8,41 @@ import (
 	"os/signal"
 	"path/filepath"
 	"strconv"
+	"sync"
 	"syscall"
+	"time"
 
+	"github.com/NethermindEth/juno/internal/db/block"
+
+	"github.com/NethermindEth/juno/internal/db/transaction"
+
+	"github.com/NethermindEth/juno/internal/db/state"
+
+	"github.com/NethermindEth/juno/internal/db/abi"
+
+	"github.com/NethermindEth/juno/internal/db/contracthash"
+	"github.com/torquem-ch/mdbx-go/mdbx"
+
+	"github.com/NethermindEth/juno/pkg/rest"
+	"github.com/NethermindEth/juno/pkg/starknet"
```

Comment:
Why is there space between imports?

Reply:
Yeah, its to do with the file watcher I used for goland. Goland support for gofumpt adds these spaces when it rearranges the imports.

Reply:
Please view the latest commit and please resolve the conversation